picknik_ur5_moveit_config.rosinstall

```diff
@@ -3,10 +3,12 @@
     local-name: ur/universal_robot
     uri: https://github.com/fmauch/universal_robot.git
     version: calibration_devel
+# TODO: revert to master when RTDE bug fixed (error "Variable 'speed_slider_mask' is currently controlled by another
+# RTDE client" on startup)
```

Comment:
UniversalRobots/Universal_Robots_ROS_Driver@117323d is a pretty old commit.

Has this been reported?

Reply:
Interesting question, I don't remember this coming up. First of all (without knowing the full context), this doesn't seem as a bug, but an intended error output. Second, without any bug report there's nothing to close.

Reply:
Yes, I'm not sure which RTDE bug is referenced in the commit comment.

I'm hoping either @tylerjw or @JStech can point us to something.

Reply:
I'm not at the robot at the moment, and I'm not an expert on this--I guess it was just received knowledge on my part. When starting the robot (say, via `roslaunch picknik_ur5_moveit_config demo.launch fake_execution:=false`), it would fail with that error message in the comment. @jliukkonen found that using 117323d let us work around that error. Maybe something in this setup needs to be updated, I don't know.

Reply:
Could I ask you (or @jliukkonen) to report this?

I probably don't have to explain to you that we can't fix what we don't know ..

Reply:
Sorry about that. I just submitted an issue: UniversalRobots/Universal_Robots_ROS_Driver#195